docs/pages/access-controls/device-trust.mdx

```diff
@@ -5,14 +5,14 @@ layout: tocless-doc
 videoBanner: gBQyj_X1LVw
 ---
 
-<Admonition type="warning">
+<Admonition type="notice">
   Device Trust supports the following components:
 
   - User devices: macOS, Windows and Linux.
   - Teleport client: `tsh` and Teleport connect.
-  - Resources: SSH, Database and Kubernetes.
+  - Resources: Apps (role-based only), SSH, Database and Kubernetes
 
-  Support for Web UI and application access are planned for upcoming Teleport
+  Support for Web UI and Desktop Access are planned for upcoming Teleport
```

Comment:
```suggestion
  Support for the Web UI and desktop access are planned for upcoming Teleport
```

We're no longer dividing Teleport into separate, resource-based products, so we don't capitalize "<Resource> Access".